### ISSUE TITLE
Feature request: see who has and hasn't solved a challenge

### ISSUE THREAD

**Issue by zardus** [pwncollege/dojo#170]
Useful for private dojos to understand class progress.

**Comment by rileyannon** [comment]
Hey all, I was taking a quick look at this one and wanted to run an idea by you. I saw two particular functions in [dojo.py](https://github.com/pwncollege/dojo/blob/628f9eb36131ae6e100983e9d2d5e7026eaec66d/dojo_plugin/pages/dojos.py#L161) that I think could be reworked into this feature: `view_dojo_activity()` and `view_dojo_solves()`.

I was thinking we could make a new template where you could get a full list of those who have finished and those who have not, either for ALL challenges (one giant combined page) or an individual page for each challenge. Since we have a list of dojo users and a list of solves, we can just find the (Total Users) \ (Solved Users) to get those who have not yet solved. 

Was this what you were thinking of in terms of this request?

**Comment by zardus** [comment]
Yeah, I think this is a great idea and lines up great for this! We can probably just also stick it right on the dojo admin page rather than making a new page for it.

If a PR implementing that appeared, I'd definitely be excited :-)
